MIDAS/src/silsim/main.cpp

```diff
@@ -18,11 +18,11 @@ Sensors create_sensors_attached_to(SimulatedRocket* sim, bool should_be_continuo
     };
 }
 
-RocketSystems* create_and_start(SimulatedRocket* attached, const char* sink_name) {
+RocketSystems* create_and_start(SimulatedRocket* attached, const char* sink_name, const char* tlm_sink) {
     RocketSystems* systems = new RocketSystems {
-            create_sensors_attached_to(attached, true),
-            {},
-            LogSink(sink_name)
+            .sensors = create_sensors_attached_to(attached, true),
+            .log_sink = LogSink(sink_name),
+            .tlm = Telemetry(TelemetryBackend(tlm_sink))
     };
 
     uint8_t* stack = new uint8_t[4096];
```

Comment:
freertos ships with type `StackType_t` and `xTaskCreateStaticPinnedToCore` takes in a `StackType_t*` so why r we not using the builtin type